code/modules/jobs/job_types/roguetown/adventurer/types/antag/brigand.dm

```diff
@@ -0,0 +1,52 @@
+/datum/advclass/brigand //Strength class, starts with axe or flails and medium armor training
+	name = "Brigand"
+	tutorial = "Cast from society, you use your powerful physical might and endurance to take from those who are weaker from you."
+	allowed_sexes = list(MALE, FEMALE)
+	allowed_races = RACES_ALL_KINDS
+	outfit = /datum/outfit/job/roguetown/bandit/brigand
+	category_tags = list(CTAG_BANDIT)
+
```

Comment:
may i suggest for the melee classes

cmode_music = 'sound/music/combat_bandit.ogg'

Reply:
while a cool track it doesn't compare to the waltzing melody of Deus Ex - Battery Park (Combat). the building tempo as it grows more frenetic, the marching beat, the ebb and flow of the horns and brass...